airbyte-webapp/src/components/base/Titles/Titles.tsx

```diff
@@ -19,13 +19,13 @@ const H1 = styled.h1<IProps>`
   margin: 0;
 `;
 
-/** @deprecated Use `<Text as="h1 | h2 | h3 | h4 | h5" size="md" />` */
+/** @deprecated Use `<Heading as="h3" />` */
```

Comment:
The comments were ok before, except for the `Text` component name change. The h3 is being misused in this implementation so when replacing a new they'd have to pick the most appropriate h type, but the size for this is equivalent to "md"

Reply:
Got it, will revert the comment except for the component name change!